web/templates/pages/members.html

```diff
@@ -20,9 +20,9 @@
 			<ul class="about_list">
 			{% for member in members_list %}
 				{% if member.activity_status == 'MC' %}
-					<li class="mentor-cat"><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Mentor cat)<li>
+					<li class="mentor-cat"><a href="{% url 'web.views.tasks' member_id=task.project_id %}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Mentor cat)<li>
```

Comment:
@fbbunny I get the following error when accessing this page:

    Reverse for 'web.views.tasks' with arguments '()' and keyword arguments '{u'member_id': ''}' not found. 1 pattern(s) tried: ['tasks/$']